docker/Dockerfile-build.jvm

```diff
@@ -5,13 +5,9 @@
 # Build the project
 FROM registry.access.redhat.com/ubi8/openjdk-11:latest AS build
 USER root
-COPY pom.xml /home/jboss
-COPY backend/src /home/jboss/backend/src
-COPY backend/pom.xml /home/jboss/backend
-COPY checkstyle/checkstyle/checkstyle.xml /home/jboss/backend
-COPY checkstyle/checkstyle/checkstyle-suppressions.xml /home/jboss/backend
+COPY . /home/jboss
 WORKDIR /home/jboss
-RUN mvn clean package -DskipTests --projects :notifications-parent,:notifications-backend
+RUN mvn clean install -DskipTests 
```

Comment:
I think install isn't needed here. I would say we don't need the dependencies to be in the local .m2/repository folder and package is enough to work with it.

my issue was that I added checktyle at the --projects flag, but in the wrong order.